boggs/src/test/java/com/hartwig/healthchecks/boggs/healthcheck/mapping/MappingExtractorTest.java

```diff
@@ -2,7 +2,7 @@
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Wrong order for 'org.junit.Assert.assertFalse' import.](https://www.codacy.com/app/hmf/health-checks/file/2780681856/issues/source?bid=3363458&fileBranchId=3365368#l5)